app/src/main/java/com/bootcamp_android/parking_app/fragments/ReservationAddFragment.kt

```diff
@@ -14,9 +17,15 @@ import com.bootcamp_android.parking_app.R
 import com.bootcamp_android.parking_app.databinding.FragmentAddReservationBinding
 import java.util.*
 
-class ReservationAddFragment : Fragment(R.layout.fragment_add_reservation) {
```

Comment:
Try to declare the binding with lateinit and clean it in onDestroy
Please do it as the following example https://github.com/android/architecture-components-samples/blob/main/ViewBindingSample/app/src/main/java/com/android/example/viewbindingsample/InflateFragment.kt

The idea is that you can use that binding variable in other methods